packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,174 @@
+import React, {
+  Children,
+  createRef,
+  isValidElement,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react"
+import styled from "styled-components"
+import { useCursor } from "use-cursor"
+import { color, space } from "../../helpers"
+import { ChevronIcon } from "../../svgs"
+import { Box, BoxProps } from "../Box"
+import { Clickable } from "../Clickable"
+import { Skip } from "../Skip"
+import { VisuallyHidden } from "../VisuallyHidden"
+import { paginate } from "./utils"
+
+const Container = styled(Box)`
+  position: relative;
+  width: 100%;
+`
+
+const Viewport = styled(Box)`
+  width: 100%;
+  overflow: hidden;
+`
+
+const Rail = styled(Box)`
+  display: flex;
+  height: 100%;
+  transition: transform 500ms;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`
+
+const ARROW_WIDTH = space(4)
+
+const Arrow = styled(Clickable)`
+  position: absolute;
+  top: 0;
+  height: 100%;
+  width: ${ARROW_WIDTH}px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  user-select: none;
+  color: ${color("black30")};
+  transition: opacity 250ms, color 250ms;
+
+  > svg {
+    fill: currentColor;
```

Comment:
Of the opinion all our SVGs should default to `currentColor`

> currentColor acts like a variable for the current value of the color property on the element. And the Cascading part of CSS is still effective, so if there’s no defined color property on an element, the cascade will determine the value of currentColor

Just let's you set color on the outer context without having to target the SVG fill explicitly.